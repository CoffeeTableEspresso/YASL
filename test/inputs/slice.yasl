##[2, 3]\n23\n[1, 2, 3, 4]\n1234\n[1, 2, 3, 4, 5]\n12345\n

const x = [1, 2, 3, 4, 5]
const y = "12345"

echo x[1:3]
echo y[1:3]

echo x[0:-1]
echo y[0:-1]
echo x[-10:20]
echo y[-10:20]
