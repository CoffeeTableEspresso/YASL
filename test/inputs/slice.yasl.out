[2, 3]
23
[1, 2, 3, 4]
1234
[1, 2, 3, 4, 5]
12345
